### PR TITLE
WebAPI: Update Method pages to modern structure (part 14)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #14857 

## Summary
Updating the method pages to follow the current templates:
https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods
https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_method_subpage_template

Changes Include:
- fix syntax section
- changing section titles to suggested titles in the template
- change `{{DOMxRef("DOMString")}}` to `string`
- remove extra spaces in between words and extra trailing spaces
- other small corrections

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/range/comparenode/index.md [hunk 1]
```diff
@@ -64,15 +64,19 @@ function rangeCompareNode(range, node) {
 ## Syntax
 
 ```js
-returnValue = range.compareNode( referenceNode );
+compareNode( referenceNode )
```
Comment (teoli2003) [inline files/en-us/web/api/range/comparenode/index.md:67]:
```suggestion
compareNode(referenceNode)
```

File: files/en-us/web/api/readablestream/getreader/index.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ While the stream is locked, no other reader can be acquired until this one is re
 ## Syntax
 
 ```js
-readableStream.getReader()
-readableStream.getReader({mode})
+getReader()
+getReader(mode)
 ```
 
 ### Parameters
```
Comment (teoli2003) [inline files/en-us/web/api/readablestream/getreader/index.md:25]: We need to remove the braces at line 27 too.

File: files/en-us/web/api/rtcpeerconnection/addicecandidate/index.md [hunk 3]
```diff
@@ -33,7 +33,8 @@ This is covered in more detail in the articles [WebRTC connectivity](/en-US/docs
 
 ```js
 addIceCandidate(candidate)
-addIceCandidate(candidate, successCallback, failureCallback) // deprecated
+addIceCandidate(candidate, successCallback)
+addIceCandidate(candidate, successCallback, failureCallback)
```
Comment (teoli2003) [inline files/en-us/web/api/rtcpeerconnection/addicecandidate/index.md:37]: I would keep the _deprected_ comment here.
```suggestion
addIceCandidate(candidate, successCallback, failureCallback) // deprecated
```

File: files/en-us/web/api/rtcpeerconnection/generatecertificate/index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: RTCPeerConnection.generateCertificate() static function
+title: RTCPeerConnection.generateCertificate()
```
Comment (teoli2003) [inline files/en-us/web/api/rtcpeerconnection/generatecertificate/index.md:2]: Noting that it is a static method and not linked to an instance is important.
```suggestion
title: RTCPeerConnection.generateCertificate() static function
```

File: files/en-us/web/api/rtcrtpreceiver/getcapabilities/index.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: RTCRtpReceiver.getCapabilities() static function
+title: RTCRtpReceiver.getCapabilities()
```
Comment (teoli2003) [inline files/en-us/web/api/rtcrtpreceiver/getcapabilities/index.md:2]:
```suggestion
title: RTCRtpReceiver.getCapabilities() static function
```

File: files/en-us/web/api/rtcrtpsender/getcapabilities/index.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: RTCRtpSender.getCapabilities() static function
+title: RTCRtpSender.getCapabilities()
```
Comment (teoli2003) [inline files/en-us/web/api/rtcrtpsender/getcapabilities/index.md:2]:
```suggestion
title: RTCRtpSender.getCapabilities() static function
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
what is the purpose of marking titles with `static function`?

**Comment by teoli2003** [comment]
> what is the purpose of marking titles with `static function`?

I don't how we will signal them in the end, but static functions are pretty different from regular ones on the prototype. It is important that devs noticed them quickly. So I think having them in the title, for now, is not too bad.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**